src/he/he_internal.h

```diff
@@ -125,6 +125,29 @@ struct he_ssl_ctx {
   he_version_info_t maximum_supported_version;
 };
 
+typedef struct he_fragment_entry_node he_fragment_entry_node_t;
+
+// Information of a fragment
+typedef struct he_fragment_entry_node {
+  uint16_t begin;
+  uint16_t end;
+  bool last_frag;
+  he_fragment_entry_node_t *next;
```

Comment:
Probably should start with the next pointer for better packing

Reply:
It doesn't make a difference, because we use `pack(1)` for all these structs.